packages/vit-s/src/shared-plot-options/CellColorEncodingOption.js

```diff
@@ -20,12 +20,13 @@ export default function CellColorEncodingOption(props) {
   }
 
   return (
-    <TableRow>
-      <TableCell className={classes.labelCell} htmlFor="cell-color-encoding-select">
+    <TableRow key="cell-color-encoding">
+      <TableCell className={classes.labelCell} variant="head" scope="row">
         {observationsLabelNice} Color Encoding
       </TableCell>
```

Comment:
```suggestion
      <TableCell className={classes.labelCell} variant="head" scope="row">
        <label htmlFor="cell-color-encoding-select">
          {observationsLabelNice} Color Encoding
        </label>
      </TableCell>
```
Maybe we should keep the `htmlFor` by using a label - although we might need to adjust the styling of the label

Reply:
I have changed this to label in all places where it was referenced.